README.md

```diff
@@ -8,42 +8,35 @@ Deblur is a greedy deconvolution algorithm for amplicon sequencing based on Illu
 
 Install
 =======
-- Deblur requires Python 3.5. If Python 3.5 is not installed, you can create a [conda](http://conda.pydata.org/docs/install/quick.html) environment for deblur using:
+- Deblur requires Python 3.5. If Python 3.5 is not installed, you can create a [conda](http://conda.pydata.org/docs/install/quick.html) environment for Deblur using:
 ```
-conda create -n deblurenv python=3 numpy
+conda create -n deblurenv python=3.5 numpy
 ```
 
 and activate it using:
 ```
 source activate deblurenv
 ```
 
-(note you will need to activate this environment every time you want to use deblur)
+(note you will need to activate this environment every time you want to use Deblur)
 
-At the moment, the install is a two stage process as we do not currently have deblur staged in a conda channel.
-
-- install deblur dependencies
+Install Deblur dependencies and Deblur itself:
 ```
-conda install -c bioconda VSEARCH MAFFT biom-format SortMeRNA==2.0
-```
-
-- Install Deblur:
-```
-pip install deblur
+conda install -c bioconda -c biocore VSEARCH MAFFT biom-format SortMeRNA==2.0 deblur
 ```
 
 Example usage
 =============
 
-The input to deblur workflow is a directory of fasta files (1 per sample) or a demultiplexed FASTA or FASTQ file. The output are three biom tables (in the output directory) with sequences as the OTU ids:
-
-- all.biom : contains all deblurred reads.
+We recommend using Deblur via the QIIME2 plugin [q2-deblur](https://github.com/wasade/q2-deblur). Examples of its use can be found within the plugin itself. However, Deblur itself does not depend on QIIME2.
 
-- reference-hit.biom : contains only deblurred reads matching the positive filtering database (default is only 16S reads)
+If you are running Deblur directly, we recommend focusing on the `workflow` subcommand. Detailed help can be obtained with:
 
-- reference-non-hit.biom : contains only deblurred reads not matching the positive filtering database (default is all non-16S reads)
+```
+deblur workflow --help
+```
 
-The simple use case just specifies the input fasta file (or directory), output directory name and the sequence trim length:
+As a simple example, let's specify an input FASTA file, an output path and a sequence trim length of 150. This command will trim all sequences in `all_samples.fna` to 150nt in length; any read that is shorter will be omitted. This execution mode assumes that `all_samples.fna` is demultiplexed such that the sequence IDs are compatible with QIIME 1.9.1. On completion, a new directory `output` will be created with multiple output files corresponding to the reference databases (see the Input and Output Files section for more detail). 
```

Comment:
remember: deblur is _not_ 16s specific. there also does not appear to be a mode of operation that is database independent. i will revise the text for clarity.